cassandra-executor/src/main/java/com/mesosphere/dcos/cassandra/executor/CassandraDaemonProcess.java

```diff
@@ -431,10 +431,17 @@ public void drain()
      * @throws ExecutionException   If execution is interrupted.
      * @throws IOException          If communication with Cassandra fails.
      */
-    public void upgradeTables()
+    public void upgradeSSTables(String keyspace, List<String> columnFamilies)
```

Comment:
If you look at the comments of this class, it says: ` All administration and monitoring is achieved by attaching to the Cassandra daemon via JMX using the NodeProbe class.`. So all NodeProbe functions are centralized to this class which has to be changed when the Cassandra API changes. 

This is the same pattern followed in Cleanup, Repair, etc.